test/unit/double_free.c

```diff
@@ -70,8 +70,30 @@ TEST_BEGIN(test_large_double_free_no_tcache) {
 }
 TEST_END
 
+TEST_BEGIN(test_double_free_tcache) {
+    test_skip_if(!config_debug);
+
+    test_large_double_free_pre();
```

Comment:
nit: Let's remove the `large` part from the _pre and _post functions names. You will also want to skip the test if tcache is disabled: https://github.com/jemalloc/jemalloc/blob/d660683d3ddc2aaebf41a5662a6bc629be016e6d/test/unit/uaf.c#L51

Reply:
addressed in [4bc176c](https://github.com/jemalloc/jemalloc/pull/2310/commits/4bc176c3002fa751f267c7a420d38a8a2df2e712)